examples/community/README.md

```diff
@@ -39,7 +39,7 @@ clip_model = CLIPModel.from_pretrained("laion/CLIP-ViT-B-32-laion2B-s34B-b79K",
 
 
 guided_pipeline = DiffusionPipeline.from_pretrained(
-    "CompVis/stable-diffusion-v1-4",
+    "runwayml/stable-diffusion-v1-5",
```

Comment:
@patil-suraj do you want to change this?

Reply:
we can change this, it should be fine here.